src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import { getArticleById } from './api/newsApi';
+import { useParams } from 'react-router-dom';
+
+const SingleArticle = () => {
+	const { article_id } = useParams();
+	const [article, setArticle] = useState({});
+	const [isLoading, setIsLoading] = useState(true);
+
+	useEffect(() => {
+		if (article_id) {
```

Comment:
Do we need an if statement here? We should only be navigating to a page that is rendering this component if we have clicked on an article and updated the url with an article id